lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu
+  
+  def initialize
```

Comment:
By including the menu hash in the initialize method, any changes to the menu within a Menu class instance would not be saved when the code was loaded up again. You would have to manually enter the changes into the menu hash for them to be permanent.